wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md

```diff
@@ -0,0 +1,45 @@
+---
+tags:
+  - medals
+  - medal hunting
+---
+
+# osu! Medal Hunters Discord server
+
+::: Infobox
+
+#### osu! Medal Hunters
+
+![Server logo](img/logo.png "The Medal Hunter Discord server logo, designed by Tanza3D")
+
+|  |  |
```

Comment:
I don't think you need this

Reply:
was copied from https://github.com/ppy/osu-wiki/issues/5440, and i think you do. at least https://markdownlivepreview.com/ doesn't consider it valid otherwise

Reply:
I misread the table, thinking the alignment is all you need. yeah totally, carry on